### PR TITLE
Set connectToDevTools for Apollo developer tools

### PR DESCRIPTION
**Pull request by revskill10**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -104,6 +104,7 @@ const ApolloClientPlugin = createPlugin({
         : [authMiddleware, connectionLink];
       const client = new ApolloClient({
         ssrMode: __NODE__ ? true : false,
+        connectToDevTools: (__BROWSER__ && __DEV__) ? true : false,
```
Comment (KevinGrandon) [inline src/index.js]: I wonder if it should just be `__BROWSER__ && __DEV__`. It does look like other defines for this object use this style though, so maybe consistency is better. Thanks for the PR!

Comment (revskill10) [reply src/index.js]: @KevinGrandon So we should remove `( )` right ?

Comment (KevinGrandon) [reply src/index.js]: Yup. I just made the commit because I saw eslint complained.

**Comment by KevinGrandon** [comment]
!merge

**Pull request merged**